spec/index.bs

```diff
@@ -2211,6 +2211,16 @@ Issue(https://github.com/gpuweb/gpuweb/issues/851): consider making `textureComp
 
         Otherwise, it must be `false`.
 
+    : <dfn>minBufferBindingSize</dfn>
+    ::
+        If the {{GPUBindGroupLayoutEntry/type}} is
+        {{GPUBindingType/"uniform-buffer"}},
+        {{GPUBindingType/"storage-buffer"}}, or
+        {{GPUBindingType/"readonly-storage-buffer"}},
+        this indicates the minimum buffer binding size.
```

Comment:
this is still optional for these types, maybe we can reword it "`this *may* contain the minimum buffer binding size`?

Reply:
Then I want to change "this contains" to "this may contain" for "hasDynamicOffset" and "multisampled" too, in order to make them consistent (at least within BindGroupLayout). Does this sound good? @kvark

Reply:
No. `hasDynamicOffset` and `multisampled` are always applicable to the relevant binding types. At the same time, `minBufferBindingSize` is optional.

Reply:
OK. Done. Thank you, @kvark